immudb/handler/get.py

```diff
@@ -26,7 +26,7 @@ def call(service: schema_pb2_grpc.ImmuServiceStub, rs: RootService, key: bytes,
     try:
         msg = service.Get(request)
     except Exception as e:
-        if hasattr(e, 'details') and e.details() == 'key not found':
+        if hasattr(e, 'details') and (e.details() == 'key not found' or e.details() == 'tbtree: key not found'):
```

Comment:
I'd suggest checking for the substring `key not found`. Next immudb release will expose error codes, making error handling easier and less error prone

Reply:
I can check with endswith() method
If i would simply check for if string is in string then we would have another backward compability issue - with expiration.

Right now in immudb-py expiration returns error instead of None, so when i would check simply for existency of "key not found" in a string - it would start to returns None
if anyone right now are using this function and opaque it in try catch - try catch would stop working

This should be handled properly and consistent, but right now is not

Reply:
#22e4a26